pkg/otlp/attributes/attributes.go

```diff
@@ -95,39 +95,77 @@ var (
 
 	// Kubernetes out of the box Datadog tags
 	// https://docs.datadoghq.com/containers/kubernetes/tag/?tab=containerizedagent#out-of-the-box-tags
+	// https://github.com/DataDog/datadog-agent/blob/d33d042d6786e8b85f72bb627fbf06ad8a658031/comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go
```

Comment:
Can we expose those constants and use them here? Am concerned about the tags diverge moving forward

Reply:
OK, probably worth leaving an issue or ticket to track it. Would be good to keep those constants in a single source of truth